scripts/linux/start_zk.sh

```diff
@@ -27,29 +27,34 @@ fi
 
 cd $INSTALL_DIR
 
-ZOOKEEPER_PKG=${PROJECT_DIR}/thirdparty/build/Download/zookeeper/zookeeper-3.4.10.tar.gz
-if [ ! -f ${ZOOKEEPER_PKG} ]; then
-    echo "no such file \"${ZOOKEEPER_PKG}\""
-    echo "please install third-parties first"
-    exit 1
+ZOOKEEPER_ROOT=apache-zookeeper-3.7.0-bin
+ZOOKEEPER_TAR_NAME=${ZOOKEEPER_ROOT}.tar.gz
```

Comment:
suggest keep consistent with the original package management: use cmake https://github.com/XiaoMi/rdsn/blob/v2.3.0/thirdparty/CMakeLists.txt#L155

Reply:
zookeeper-3.4.10 just have one package for us. (https://zookeeper.apache.org/releases.html) But on zookeeper-3.7.0 it provided two packages, one just has bin what we can use it to start zookeeper cluster, another one both bin and source code.  I think we just need bin when we need start zookeeper cluster.

Reply:
Using Cmake is good for package management, and also we can use OSS to speed up download